api/fastpm/powerspectrum.h

```diff
@@ -57,4 +57,25 @@ fastpm_powerspectrum_scale(FastPMPowerSpectrum * ps, double factor);
 void
 fastpm_powerspectrum_rebin(FastPMPowerSpectrum * ps, size_t newsize);
 
+typedef struct FastPMFuncK {
```

Comment:
This is pretty cool!

Looks like the class works for any linearly interpolated tabulated function. But too eagerly adapting an over general name is bad too.